ci-operator/step-registry/enable-qe-catalogsource/disconnected/enable-qe-catalogsource-disconnected-commands.sh

```diff
@@ -251,13 +251,14 @@ function check_marketplace () {
     #     echo "marketplace installed, skip..."
     #     return 0
     # fi
-    
+    set +e
```

Comment:
@jianzhangbjz I am not sure if there is side effective to close `-e`, but I have another way to resolve this issue.
change
```shell
run_command "oc get ns openshift-marketplace"; ret=$?
```
to
```shell
ret=0
run_command "oc get ns openshift-marketplace" || ret=$?
```
could resolve your issue.

FYI.

Reply:
agree with @kuiwang02 , or you can move the command into if condition:

`if $(run_command "oc get ns openshift-marketplace" 2 >/dev/null ; then`

Reply:
Great! Updated it, thanks!